src/sessions/SessionsSection.tsx

```diff
@@ -53,6 +90,25 @@ export const SessionsSection = () => {
           loader={loader}
           ariaLabelKey="session:title"
           searchPlaceholderKey="sessions:searchForSession"
+          searchTypeComponent={
+            <Select
+              width={200}
+              data-testid="filter-session-type-select"
+              isOpen={filterDropdownOpen}
+              className="filter-session-type-select"
+              variant={SelectVariant.single}
+              onToggle={() => setFilterDropdownOpen(!filterDropdownOpen)}
+              toggleIcon={<FilterIcon />}
+              onSelect={(_, value) => {
+                setFilterType(value as string);
```

Comment:
Instead of type-casting value here we can also call `toString()` instead, as this is a method that any `SelectOptionObject` [must implement](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Select/SelectOption.tsx#L12).
```suggestion
                setFilterType(value.toString());
```

Reply:
Done